gremlin-groovy/src/test/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GremlinGroovyScriptEngineTest.java

```diff
@@ -474,122 +467,6 @@ public void customizerShouldOverrideCoreImports() throws Exception {
-        assertEquals(10, bytecodeBindings.size());
-        assertThat(bytecodeBindings.containsKey("x"), is(true));
-        assertThat(bytecodeBindings.containsKey("y"), is(true));
-        assertThat(bytecodeBindings.containsKey("v1Id"), is(true));
-        assertThat(bytecodeBindings.containsKey("l"), is(true));
-        assertThat(bytecodeBindings.containsKey("o"), is(true));
-        assertThat(bytecodeBindings.containsKey("m1"), is(true));
-        assertThat(bytecodeBindings.containsKey("m2"), is(true));
-        assertThat(bytecodeBindings.containsKey("m3"), is(true));
-        assertThat(bytecodeBindings.containsKey("two"), is(true));
-        assertThat(bytecodeBindings.containsKey("three"), is(true));
-
-        assertEquals("g.V(v1Id).has('person','age',29).has('person','active',x).in_('knows').merge_e(m1).merge_v(m2).option(Merge.on_create,m3).merge_v(__.identity()).choose(__.out().count()).option(two,__.name).option(three,__.age).filter_(__.outE().count().is_(y)).map(l).order().by('name',o)", gremlinAsPython);
-    }
-
-    @Test
-    public void shouldHandleNaNInf() throws Exception {
-        final GremlinGroovyScriptEngine engine = new GremlinGroovyScriptEngine((GroovyCustomizer)
-                () -> new RepeatASTTransformationCustomizer(new VarAsBindingASTTransformation()));
-
-        final GraphTraversalSource g = traversal().withEmbedded(EmptyGraph.instance());
-        final Bindings bindings = new SimpleBindings();
-        bindings.put("g", g);
-
-        final Traversal.Admin t = (Traversal.Admin)
-                engine.eval("g.inject(-Infinity,NaN,xx1).is(P.eq(Infinity).or(P.eq(NaN)).or(P.eq(+Infinity)))", bindings);
-        final GremlinLang bytecode = t.getGremlinLang();
-
-        final Map<String,Object> bytecodeBindings = bytecode.getBindings();
-        assertEquals(1, bytecodeBindings.size());
-        assertThat(bytecodeBindings.containsKey("xx1"), is(true));
-
-        final JavascriptTranslator jsTranslator = JavascriptTranslator.of("g");
-        final String gremlinAsJs = jsTranslator.translate(bytecode).getScript();
-        assertEquals("g.inject(Number.NEGATIVE_INFINITY,Number.NaN,xx1).is(P.eq(Number.POSITIVE_INFINITY).or(P.eq(Number.NaN)).or(P.eq(Number.POSITIVE_INFINITY)))", gremlinAsJs);
-
-        final PythonTranslator pyTranslator = PythonTranslator.of("g");
-        final String gremlinAsPy = pyTranslator.translate(bytecode).getScript();
-        assertEquals("g.inject(float('-inf'),float('nan'),xx1).is_(P.eq(float('inf')).or_(P.eq(float('nan'))).or_(P.eq(float('inf'))))", gremlinAsPy);
-
-        final DotNetTranslator dnTranslator = DotNetTranslator.of("g");
-        final String gremlinAsDn = dnTranslator.translate(bytecode).getScript();
-        assertEquals("g.Inject(Double.NegativeInfinity,Double.NaN,xx1).Is(P.Eq(Double.PositiveInfinity).Or(P.Eq(Double.NaN)).Or(P.Eq(Double.PositiveInfinity)))", gremlinAsDn);
-    }
-
-    @Test
```

Comment:
it occurs to me that we don't have this kind of coverage anywhere for the new translators? are we losing some coverage by removing this test? should we add some coverage somewhere else to make up fo rit?

Reply:
i meant particularly for `call()`

Reply:
I added test case for translator and [ticket](https://issues.apache.org/jira/browse/TINKERPOP-3091) to track feature tests